File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--changing-worksheet-formatting-using-checkbox-controls_3.cs
        private void applyUnderlineFont_Click(object sender, EventArgs e)
        {
            if (this.applyUnderlineFont.Checked == true)
            {
                this.textFont.Font.Underline = Excel.XlUnderlineStyle.xlUnderlineStyleSingle;
            }
            else
            {
                this.textFont.Font.Underline = Excel.XlUnderlineStyle.xlUnderlineStyleNone;
            }
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--changing-worksheet-formatting-using-checkbox-controls_3.vb
    Private Sub applyUnderlineFont_Click(ByVal sender As Object, ByVal e As System.EventArgs) _
        Handles applyUnderlineFont.Click

        If Me.applyUnderlineFont.Checked Then
            Me.textFont.Font.Underline = Excel.XlUnderlineStyle.xlUnderlineStyleSingle
        Else
            Me.textFont.Font.Underline = Excel.XlUnderlineStyle.xlUnderlineStyleNone
        End If
    End Sub